docs/docs/installation/docker-compose.md

```diff
@@ -55,6 +55,10 @@ Mathesar actually uses two databases; one internal system database for metadata
 
 The credentials created in this section are used to login directly to the database (i.e., not through the Mathesar UI).
 
+**Note on permissions**: If you choose to connect Mathesar to a preexisting PostgreSQL database, you need to make sure that the user you choose:
```

Comment:
I'd like to make additional documentation updates, but I can handle those myself tomorrow.